Modules/Core/ImageAdaptors/test/itkComplexConjugateImageAdaptorTest.cxx

```diff
@@ -34,13 +35,13 @@ itkComplexConjugateImageAdaptorTest(int, char *[])
   image->Allocate();
   const ImageType::RegionType region = image->GetLargestPossibleRegion();
 
-  srand(50L);
+  std::mt19937                          randomNumberEngine;
+  std::uniform_real_distribution<float> randomNumberDistribution;
 
   itk::ImageRegionIterator<ImageType> iter(image, region);
   for (iter.GoToBegin(); !iter.IsAtEnd(); ++iter)
   {
-    auto            randMax = static_cast<float>(RAND_MAX);
-    const PixelType pixel(static_cast<float>(rand()) / randMax, static_cast<float>(rand()) / randMax);
+    const PixelType pixel(randomNumberDistribution(randomNumberEngine), randomNumberDistribution(randomNumberEngine));
```

Comment:
For the record, this pull request was inspired by a discussion we had before about the use of `static_cast<float>(RAND_MAX)`, at https://github.com/InsightSoftwareConsortium/ITK/pull/5337#issuecomment-2847162171 It appeared that `RAND_MAX` is platform dependent, so this `static_cast<float>` may or may not be lossless. Which made me want to get rid of `RAND_MAX` 👹 !